src/pages/marketplace/index.js

```diff
@@ -14,9 +15,13 @@ const Marketplace = ({
   ...props
 }) => {
   const router = useRouter();
-
   return (
     <>
+      <Head>
+        <title>{props.title}</title>
+        <meta property="og:title" content={props.title} />
+        <meta property="og:description" content={props.description} />
```

Comment:
Each toJSON object has a meta property, which contains actual meta for meta description and meta title. Please inspect the incoming object and use that. For example `props.meta.web. seo_meta_description `
![image](https://user-images.githubusercontent.com/729972/169875143-9700ef89-de13-432d-9403-17553250e750.png)